src/mango_native_proc.erl

```diff
@@ -227,6 +238,10 @@ get_default_enabled(Props) ->
     end.
 
 
+get_index_array_lengths(Props) ->
+    couch_util:get_value(<<"index_array_lengths">>, Props, true).
```

Comment:
Ah, moving the check to here is fine. But we're still doing input validation. So it'd be returning true/false if the value is true vs anything else.